src/lib/connectors/cdp/cdp-launcher.ts

```diff
@@ -31,7 +31,9 @@ export class CDPLauncher extends Launcher {
 
         chromeFlags.push('--no-default-browser-check');
         if (isCI) {
-            chromeFlags.push('--headless', '--disable-gpu');
+            // Running in `headless` mode is causing error in tests for `set-cookie` header.
+            // So the `--headless` flag was removed.
```

Comment:
> and `--headless` breaks the tests for the cookies rules...

Removing it [will break the test for quite a few other rules](https://travis-ci.org/sonarwhal/sonar/jobs/263597186). :(

Maybe there is another way?